src/components/Card.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
-import cx from 'classnames'
+import classnames from 'classnames'
 
 import './card.scss';
 
 const Card = ({ card, onClick, isFlipped, inactive }) => {
   const { type, image } = card;
 
   return (
-    <div className={cx('Card', { inactive })} >
+    <div className={classnames('Card', { inactive })} >
       <div
-        className={cx('Card_cardContainer', { isFlipped })}
+        className={classnames('Card__container', { isFlipped })}
         onClick={onClick}
       >
-        <div className="Card_face Card_face--front" aria-label={type} />
-        <div className="Card_face Card_face--back">{image}</div>
+        <div className="cardFace front" aria-label={type} />
+        <div className="cardFace back">{image}</div>
```

Comment:
I actually like what you had before better. With sass you actually want to have as few layer as possible so being specific will help decrease the layer of css in sass. Also if another component is using `front` class and apply `!important` then this class most likely will be affected as well.